### PR TITLE
Remove validator.go inside docs

### PR DESCRIPTION
**Pull request by kannappanr**
Existing code cannot be used as is.
Needs to be rewritten. OK to remove it now.

### CODE REVIEW

**Comment by poornas** [comment]
@kannappanr , can you check travis failure?

**Comment by harshavardhana** [comment]
change `Makefile: checks:` @kannappanr

File: Makefile [hunk 1]
```diff
@@ -13,8 +13,5 @@ test:
 examples:
 	@mkdir -p /tmp/examples && for i in $(echo examples/s3/*); do go build -o /tmp/examples/$(basename ${i:0:-3}) ${i}; done
 
-docs:
```
Comment (harshavardhana) [inline Makefile]: We should change in `checks: vet test examples docs functional-test` here as well

Comment (kannappanr) [reply Makefile]: done.

**Pull request merged**